packages/core/test/combinator/multicast-test.js

```diff
@@ -0,0 +1,49 @@
+import { describe, it } from 'mocha'
+import { assert, eq } from '@briancavalier/assert'
+import { spy } from 'sinon'
+
+import { now } from '../../src/source/now'
+import { map, tap } from '../../src/combinator/transform'
+import { multicast } from '../../src/combinator/multicast'
+import { runEffects } from '../../src/runEffects'
+import { ticks } from '../helper/testEnv'
+
+describe('multicast', () => {
```

Comment:
Im guessing for something like most-subect we should export MulticastSource?

will port over the other tests into multicast-test.js for the time being